pyproject.toml

```diff
@@ -6,12 +6,14 @@ readme = "README.md"
 authors = ["Matthew Richards <matthew.richards@stfc.ac.uk>"]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"
+python = ">=3.8,<3.12"
 fastapi = "^0.103.2"
 motor = "^2.5.1"
 uvicorn = "^0.20.0"
 python-multipart = "^0.0.5"
 PyYAML = "^6.0"
+ImageHash = "4.3.1"
+scipy = "1.10.0"
```

Comment:
Could you add a comment above the `scipy` line to explain that this is a dependency of `ImageHash` but is listed in our `pyproject.toml` to mitigate against a vulnerability in `scipy` version 1.9.3?